tree/ntuple/v7/src/RMiniFile.cxx

```diff
@@ -1172,6 +1199,8 @@ ROOT::Experimental::Internal::RMiniFileReader::GetNTupleProper(std::string_view
       return R__FAIL("RNTuple anchor checksum mismatch");
    }
 
+   fMaxBlobSize = ntuple->fMaxKeySize;
```

Comment:
Looking at this: if we read a version 4 RNTuple (without max key size) in the mini file, wouldn't we read the checksum as max key size?

Reply:
True; how should be go about this? Should we declare version 4 unsupported or have some sort of manual schema evolution in RMiniFile?